contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -152,6 +155,10 @@ contract LidoOracle is ILidoOracle, AragonApp {
         _tryPush();
     }
 
+    function setQuorumDelegate(address _addr, bytes4 _sig) external /* TODO: auth? */ {
```

Comment:
only by voting

Reply:
we can require function sig to be constant (e.g. "processLidoOracleReport") to simplify the code and interfaces

Reply:
yes, sounds good, will do